ext/rust-openssl

```diff
@@ -1 +0,0 @@
-Subproject commit d766671bda3777690ef71739bbdd29382d56a7bc
```

Comment:
Probably yeah. Whoops... not sure I have time to look at this for a while.

On Sun, Jan 18, 2015 at 2:06 PM, Richo Healey <notifications@github.com>
wrote:

> In ext/rust-openssl
> <https://github.com/richo/stripe-rust/pull/1#discussion-diff-23137537>:
>
> > @@ -1 +0,0 @@
> > -Subproject commit d766671bda3777690ef71739bbdd29382d56a7bc
>
> It seems like these would need a Cargo.toml and related port to hyper?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/richo/stripe-rust/pull/1/files#r23137537>.
>